qa-deploy

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
+
+# Bash strict mode
+set -euo pipefail
+
+USAGE="Usage
+===
+
+  $ ./qa-deploy PRODUCTION_URL [STAGING_URL]
+
+Description
+---
+Deploy locally to minikube.
+"
+
+function invalid() {
+    message=${1}
+    echo "Error: ${message}"
+    echo ""
+    echo "$USAGE"
+    exit 1
+}
+
+function run_minikube() {
+    # Run minikube
+    if ! minikube ip &> /dev/null; then
+        minikube start
+    fi
+
+    minikube addons enable ingress
+    kubectl config set-context minikube
+}
+
+function set_up_environment() {
+    kubectl apply -f config-maps/proxy.config.yaml
+    kubectl apply -f namespaces.yaml
```

Comment:
I don't think we should apply namespaces, as we talked about not using namespaces for local QA.

If we want to apply the PodPreset, we should probably separate the file. I don't think we need to apply it right now though.
Maybe the alternative is we create a `qa` namespace.